components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -170,22 +170,24 @@ symbols!(
 #[cfg(all(test, feature = "datagen"))]
 mod test {
     use super::*;
+    use tinystr::tinystr;
 
     fn serialize() -> Vec<u8> {
-        let months = months::SymbolsV1([
-            Cow::Owned("January".to_string()),
-            Cow::Owned("February".to_string()),
-            Cow::Owned("March".to_string()),
-            Cow::Owned("April".to_string()),
-            Cow::Owned("May".to_string()),
-            Cow::Owned("June".to_string()),
-            Cow::Owned("July".to_string()),
-            Cow::Owned("August".to_string()),
-            Cow::Owned("September".to_string()),
-            Cow::Owned("October".to_string()),
-            Cow::Owned("November".to_string()),
-            Cow::Owned("December".to_string()),
-        ]);
+        let months = [
+            (&MonthCode(tinystr!(4, "M01")), "January"),
```

Comment:
I think we should have a more convenient way of constructing month codes. How about a monthcode macro?

Reply:
I still sort-of think that MonthCode should enforce the syntax, `/^M\d\dL?$/`, in which case a macro makes sense. But if there's no invariant, I don't think there's much of a win.

Reply:
I feel like perhaps that's a followup worth filing for polish, then. There may be some discussion necessary to see if we'll ever expand month code syntax with other things.